mozilla_django_oidc/auth.py

```diff
@@ -45,6 +45,11 @@ def __init__(self, *args, **kwargs):
         self.OIDC_OP_USER_ENDPOINT = import_from_settings('OIDC_OP_USER_ENDPOINT')
         self.OIDC_RP_CLIENT_ID = import_from_settings('OIDC_RP_CLIENT_ID')
         self.OIDC_RP_CLIENT_SECRET = import_from_settings('OIDC_RP_CLIENT_SECRET')
+        self.OIDC_RP_SIGN_ALGO = import_from_settings('OIDC_RP_SIGN_ALGO', 'HS256')
+        self.OIDC_RP_IDP_SIGN_KEY = import_from_settings('OIDC_RP_IDP_SIGN_KEY', None)
```

Comment:
No, because you still need a CLIENT_SECRET that is different from the signing key.
The client secret that is provided is used for exchanging an authorization code to an access token in the code flow, but the id_token is signed with a different (provider) key.